src/libcsg/topologyreader.cc

```diff
@@ -1,5 +1,5 @@
 /* 
- * Copyright 2009-2011 The VOTCA Development Team (http://www.votca.org)
+ * Copyright 2009-2017 The VOTCA Development Team (http://www.votca.org)
```

Comment:
If you don't touch the code, don't change it!

Reply:
Undo.

Reply:
Does it matter if I just change it back as I have. Or do you want me to go back to the commit where this was made and remove it from the commit history. 

o - > x - > o    =     o      x      o
                                   \ _____/

Reply:
Just undo it for now, we can later decide it want to squash all commits of this PR into one depending on how much it is in total!